test/image-blueprints/layer5-bootc/group0/rhel94-ostree-microshift-previous-minor.container-encapsulate

```diff
@@ -0,0 +1 @@
+rhel-9.4-microshift-4.16
```

Comment:
Note that the build_bootc_images.py script does not support templating in container files yet.
We can implement it in another PR and change this and few other locations to make them more dynamic.
See https://issues.redhat.com/browse/USHIFT-3764